.eslintrc

```diff
@@ -1,30 +1,30 @@
 {
-    "env": {
-        "browser": true,
-        "es2021": true
+  "env": {
+    "browser": true,
```

Comment:
Does this look like an indent size of 2 to you? I thought you updated the indent size to 4

Reply:
I think 2 is fine, that is the previous state for this repo right?

Reply:
yes